### PR TITLE
Update readme

### LINKED ISSUE #14: Wine - Add Config Variable to Heroku
- [x] Add the wine api key to Heroku

### PR DESCRIPTION
**Pull request by Diana20920**
### What Changed?
- Initial structure has been added to the README, deals with #4 
- Added the `WINE_KEY` to Heroku app, closes #14 

### What's Next?
- Will look at reorganizing the file tree to match the weather microservice
- Will address rubocop offenses

### Known Issues?
- All local tests still passing

### Is Rubocop Happy?
- `13 files inspected, 96 offenses detected, 80 offenses auto-correctable`

### README & Documentation
- I have used the READ me structure from FE to start this one. 
- Used an Atom package to add an image of our SOA
- Added links to the repos and Heroku apps for FE, BE, and Weather Microservice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,88 @@
...
+### Prerequisites
+
+To run this application you will need Ruby 2.5.3 and Rails 5.2.5
+
+### Installing
+
+- Install the gem packages  
+`bundle install`
+
+- Create the database by running the following command in your terminal
```
Comment (klatour324) [inline README.md]: Will we need the user to run this at all since there's no db for the microservice?

File: README.md [hunk 2]
```diff
@@ -1 +1,88 @@
...
+To get the web application running, please fork and clone down the repo.
+`git clone <your@github.account:WeatherVine/front_end.git>`
+
+### Prerequisites
+
+To run this application you will need Ruby 2.5.3 and Rails 5.2.5
+
+### Installing
+
+- Install the gem packages
```
Comment (klatour324) [inline README.md:31]: With the `Figaro` gem, they may also need to run a `bundle exec figaro install` to generate the `application.yml` file so they can store the api key and then `.gitignore` that file

File: README.md [hunk 3]
```diff
@@ -1 +1,88 @@
...
+## Endpoints
+
+## Built With
+- Sinatra
+- Ruby
+- RSpec
+
+
+## Versioning
+
```
Comment (klatour324) [inline README.md:68]: Do we know what version of Sinatra we are installing? We could also add for versioning purposes what version of Ruby has ben used and rbenv here maybe?

File: README.md [hunk 4]
```diff
@@ -1 +1,88 @@
...
+## Service Oriented Architecture
+![](assets/README-f315b6d0.png)
+
+## Other Repos
+- Weather Vine Front-End Application
+  - [Repo](https://github.com/WeatherVine/front_end) | [Heroku](https://weathervine-fe.herokuapp.com/)
+- Weather Vine Back-End Application
+  - [Repo](https://github.com/WeatherVine/back_end) | [Heroku](https://weathervine-be.herokuapp.com/)
+- Weather Vine Weather Microservice
+  - [Repo](https://github.com/WeatherVine/weather_service) | [Heroku](https://weather-service-sinatra.herokuapp.com/)
```
Comment (klatour324) [inline README.md:50]: I like this section! It keeps everything connected. Good add! I will probably take it for the FE and BE READMEs.

**Review by klatour324 — COMMENTED** [review]
Great work here Diana! Take a look at a couple of the comments I left, otherwise it's looking like it's coming together!

File: README.md (hunk 1 above)
Comment (Diana20920) [reply README.md]: I definitely had questions about this. I was leaning to no so I will remove that section, TY 👍🏼

**Comment by Diana20920** [comment]
TY @klatour324, those were very helpful and I have updated the README. However we can hold on merging this PR until the README is finalized 😎

**Pull request merged**